.github/ISSUE_TEMPLATE/new-environment.md

```diff
@@ -55,18 +55,34 @@ HQ,HMPPS,OPG,LAA,HMCTS,CICA,Platforms,CJSE,Probation
 
 ### Connectivity
 
+#### Subnet sets
+
+<!--Please choose one of the below, most applications will use the general subnet set for their business unit. This means that they will benefit from out of the box connectivity to other applications, most applications will use the general subnet.  If an application has highly sensitive data it may need to go into a subnet with limited connectivity. -->
+
+- [ ] General - this application will use the same subnet as other applications in your business unit
+- [ ] Isolated - this application has highly sensitive data which must have its own isolated subnet
+
+#### How do users connect to the application?
+
+- [ ] Over the public internet
+- [ ] With a purple cabled device (please give details)
+- [ ] With a MoJ Official device
+
+#### Connectivity to other applications or external parties
+
 <!-- Please detail here and connectivity that your application needs, eg to other applications or external parties -->
 
-### Additional features
+## Additional features
 
 <!-- 
 Please check any additional features required. For more information see here
 https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/creating-networking.html#certificate-services
+If you are not sure you can leave these blank and they can be added at a later date
 -->
 
-- [ ] Bastion
```

Comment:
This was just because it's a module that can be added at any point in time now, originally it was in here as we had it as an option on the environment json file (the option is still there and needs removing actually)